cypress.json

```diff
@@ -4,9 +4,7 @@
     "testUser": "browsertestuser",
     "testUserMail": "browsertestuser@example.com",
     "testPwd": "browsertestuser",
-    "changePwdUserMail": "changePwdUser@example.com",
-    "changePwdPwd": "changePwdPwd",
-    "newPwd": "newPwd"
+    "newPwd": "browsertestuser"
```

Comment:
Perhaps would make it clearer if in the test we're just using 'testPwd' too? (to at least be 'honest' that it's only testing a no-op password change)

Reply:
Done.